src/shogun/classifier/svm/LibLinear.cpp

```diff
@@ -56,6 +56,7 @@ void CLibLinear::init()
 	C2=1;
 	set_max_iterations();
 	epsilon=1e-5;
+	set_compute_bias(false);
```

Comment:
From a quick look , I think because bias calculation is considered in LibLinear::train_machine() and is set internally.